ProcessMaker/Jobs/StartEventConditional.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace ProcessMaker\Jobs;
+
+use ProcessMaker\Models\Process as Definitions;
+
+class StartEventConditional extends BpmnAction
+{
+    public $definitionsId;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Definitions $definitions)
```

Comment:
The constructor parameter is not documented

Reply:
fixed